docs/examples/data-completeness.py

```diff
@@ -0,0 +1,85 @@
+"""
+Missing Data Periods
+===================
```

Comment:
These title underlines needs to be the same length as the text, otherwise sphinx clutters up the build log complaining about it.

Reply:
Updated in 
commit 839c794a37982d4b12951cbb283d33774f27bb03